spec/helpers/sufia_helper_spec.rb

```diff
@@ -186,4 +186,16 @@ def create_models(model, user1, user2)
       conn.commit
     end
   end
+
+  describe "#is_url?" do
+    it "returns false for non-url string" do
+      url = 'all rights reserved'
+      expect(helper.is_url?(url)).to be_falsey
```

Comment:
can you switch this to `be false` This indicates the method returns a true boolean value which is more specific than `falsey`.